src/Tizen.NUI/src/public/Layouting/FlexLayout.cs

```diff
@@ -570,11 +570,16 @@ public enum FlexJustification
             /// </summary>
             SpaceBetween,
             /// <summary>
+            /// Items are positioned with equal space before, and after the lines.<br/>
+            /// </summary>
+            SpaceAround,
+            /// <summary>
             /// Items are positioned with equal space before, between, and after the lines.<br/>
-            /// Compared to <see cref="FlexJustification.SpaceBetween"/> using <see cref="FlexJustification.SpaceAround"/>
+            /// Compared to <see cref="FlexJustification.SpaceBetween"/> using <see cref="FlexJustification.SpaceEvenly"/>
```

Comment:
I don' get it.  Did you mean "Compared to (sth else) will result ...", then the result described should be description of this one. If you want to describe the other one, then rather use "which will result ...". Or did you meant "Similar to (sth else) ", but then the detailed description of the other one should rather go there.
```suggestion
            /// Similar to <see cref="FlexJustification.SpaceBetween"/>. Using <see cref="FlexJustification.SpaceEvenly"/>
```

Reply:
Previously, I simply changed the word SpaceAround to SpaceEvenly.
(Because the documentation for SpaceAround was actually for SpaceEvenly.)

Now, I have updated the documentation for SpaceEvenly to make users understand clearly.